The linker on Darwin/x86 needs archives with an index, so create one.

Ported-from: files/Net-DNS-0.64-ar.patch

diff -Nuar Net-DNS-0.68.orig/Makefile.PL Net-DNS-0.68/Makefile.PL
--- Net-DNS-0.68.orig/Makefile.PL	2012-01-27 23:02:34.000000000 +0000
+++ Net-DNS-0.68/Makefile.PL	2012-02-29 09:03:14.939727214 +0000
@@ -336,7 +336,7 @@
 use Config qw(%Config);
 
 sub postamble {
-	my $cr = ( $^O eq 'MSWin32' && $Config{'cc'} eq 'cl' ? '/OUT:' : 'cr ' );    # ar action
+	my $cr = ( $^O eq 'MSWin32' && $Config{'cc'} eq 'cl' ? '/OUT:' : 'crs ' );    # ar action
 
 	my $content = '
 test_cover : pure_all
